tnb/validators.py

```diff
@@ -20,7 +20,7 @@ def fetch_account_balance(self, account_number):
 
         return self.fetch(f"/accounts/{account_number}/balance")
 
-    def fetch_account_balance_lock(self, account_number):
+    def fetch_account_balance_lock(self, account_number: str) -> object:
```

Comment:
This method returns a `dict`.

**Note**: The method documentation is also incorrect. 

```suggestion
    def fetch_account_balance_lock(self, account_number: str) -> dict:
```